Yoast/javascript
# Add specs for Researcher and getStemmer

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds unit tests for the Czech Researcher and the Czech getStemmer file to confirm that the keyphrase and readability assessments work as expected.

## Relevant technical choices:
 

## Test instructions for Lingo team

This PR can be tested by following these steps:

* Comment out lines of the code that import the morphology data file and test the stemmer (they and getStemmerSpec are failing until this PR is merged and the morphology data is available)
* Run yarn test for the researcherSpec file.
* If the test for `"returns false if the default research is deleted in the Czech Researcher"` fails, make sure that [this PR](https://github.com/Yoast/javascript/pull/1101/files) is merged and that the feature branch is pulled from prior to testing.


## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/secure/RapidBoard.jspa?rapidView=80&projectKey=LINGO&modal=detail&selectedIssue=LINGO-699
